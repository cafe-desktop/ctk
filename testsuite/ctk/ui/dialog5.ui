<!-- invalid dialog markup element -->
<interface>
  <object class="CtkDialog">
    <child type="action">
      <object class="CtkButton" id="button_ok">
        <property name="can-default">True</property>
      </object>
    </child>
    <action-widgets>
    </action-widgets>
    <warbl/>
  </object>
</interface>
