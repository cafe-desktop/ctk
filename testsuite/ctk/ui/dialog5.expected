ERROR: ctk-builder-error-quark 1
<input>:11:13 Unhandled tag: <warbl>
